tests/plugins/toolsets/datadog/rds/test_datadog_rds_live.py

```
@@ -0,0 +1,103 @@
+
+These tests require valid Datadog API credentials and access to RDS instances.
+Set the following environment variables:
+- DD_API_KEY: Datadog API key
+- DD_APP_KEY: Datadog Application key
+- DD_SITE_API_URL: Datadog site API URL (e.g., https://api.datadoghq.com)
+- DD_TEST_RDS_INSTANCE: RDS instance identifier to test with
+"""
+
+import os
+import pytest
+from datetime import datetime, timezone, timedelta
```

Ruff (0.12.2)

`datetime.datetime` imported but unused

Remove unused import

(F401)